tests/integration/tier0/backup-release/test_backup_release.py

```diff
@@ -29,7 +29,7 @@ def test_backup_os_release_no_envar(shell, convert2rhel):
     assert shell("mv /etc/yum.repos.d/* /tmp/s_backup/").returncode == 0
 
     # EUS version use hardoced repos from c2r as well
-    if "centos-8" in system or "oracle-8.4" in system:
+    if "centos-8" in system:
```

Comment:
Oracle Linux doesn't have any hardcoded repos anymore.

Reply:
Is this something you're saying needs to change in the PR? Or just explaining why it's removed

Reply:
Just explaining.